File: Data Structures & Algorithms/is-anagram/submission-2.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        a={}
        b={}
        for i in s:
            if i in a:
                a[i]=a[i]+1
            else :
                  a[i]=1
        for i in t:
            if i in b:
                b[i]=b[i]+1
            else :
               b[i]=1
        if a==b:
            return True
        return False

File: Data Structures & Algorithms/length-of-last-word/submission-2.java
class Solution {
    public int lengthOfLastWord(String s) {
         int index=s.lastIndexOf(" ");
        int len=s.length();
        if (index==len-1)
        {
        s=s.trim();
        int index2=s.lastIndexOf(" ");
        int len2=s.length();
        int length2=0;
        for(int i=index2+1;i<len2;i++)
        {length2=length2+1;
        }
        return length2;
        }
        int length=0;

        for(int i=index+1;i<len;i++)
        {length=length+1;
        }
        return length;
        
    }
}